mu-plugins/blocks/local-navigation-bar/postcss/style.pcss

```diff
@@ -123,6 +123,13 @@
 	}
 }
 
+@media (min-width: 890px) {
+	/* stylelint-disable selector-id-pattern */
+	#wp--skip-link--target {
+		scroll-margin-top: var(--wp--custom--local-navigation-bar--spacing--height, 0);
+	}
+}
+
```

Comment:
Moved from the sidebar container styles, as it relates directly to the change in positioning of the local nav bar